image_topic: /camera/color/image_raw
cameraMatrix: 135.61, 0, 949.049,0, 1358.42, 535.123,0, 0, 1
disCoeffs: 0, 0, 0, 0